src/commonTest/groovy/mpern/sap/commerce/build/ExtensionsTestUtils.java

```diff
@@ -0,0 +1,88 @@
+package mpern.sap.commerce.build;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import java.util.Locale;
+import java.util.stream.Stream;
+
+import static mpern.sap.commerce.build.HybrisPlugin.HYBRIS_BIN_DIR;
+
+/**
+ * Utility functions to generate extensions for the test.
+ */
+public final class ExtensionsTestUtils {
+
+    public static void generateExtension(Path location, String extensionName, List<String> dependencies)
+        throws IOException {
+
+        Path extensionFolder = location.resolve(extensionName);
+        Files.createDirectories(extensionFolder);
+        Path extensionInfoFile = extensionFolder.resolve("extensioninfo.xml");
+        Files.createFile(extensionInfoFile);
+
+        String firstUpExtensionName = extensionName.substring(0, 1).toUpperCase(Locale.ROOT) + extensionName.substring(1);
+
+        StringBuilder dependenciesContent = new StringBuilder();
+        for (String dependency : dependencies) {
+            dependenciesContent.append("<requires-extension name=\"").append(dependency).append("\"/>").append(System.lineSeparator());
+        }
+
+        String extensionInfoContentPattern =
+            "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\"?>%n"
+                + "  <extensioninfo xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:noNamespaceSchemaLocation=\"extensioninfo.xsd\">%n"
+                + "    <extension abstractclassprefix=\"Generated\" classprefix=\"%2$s\" managername=\"%2$sManager\"%n"
```

Comment:
It’s considered bad practice to use a custom manager class. Will this generated config be used in the project or just for resolving during setup?

Reply:
This is just generating a demo xml file for the tests to parse, nothing else.